images/router/haproxy/reload-haproxy

```diff
@@ -53,7 +53,7 @@ function haproxyHealthCheck() {
   while true; do
     local httpcode=$(curl $timeout_opts -s -o /dev/null -I -H "Host: " -w "%{http_code}" ${url})
 
-    if [ "$httpcode" == "503" ]; then
+    if [[ "$httpcode" == *" 503 "* || "$httpcode" == *" 404 "* ]]; then
```

Comment:
Why do you need globs?
```suggestion
    if [[ "$httpcode" = 503 || "$httpcode" = 404 ]]; then
```

Reply:
Still `==` not `=`, right?

Reply:
`=` is traditional POSIX, but Bash allows `==` as well.  My preference is `=`, but `==` should be fine if there is a strong preference for it.